warehouse/observations/tasks.py

```diff
@@ -65,6 +65,13 @@ def report_observation_to_helpscout(task, request: Request, model_id: UUID) -> N
     # Fetch the Observation from the database
     model = request.db.get(Observation, model_id)
 
+    # Check to see if this ObservationKind should be sent
+    if model.kind not in [
+        ObservationKind.IsDependencyConfusion.value[0],
+        ObservationKind.IsMalware.value[0],
```

Comment:
Optimize: I think you can use the `OBSERVATION_KIND_MAP` here instead of value access, similar to
https://github.com/pypi/warehouse/blob/c41aa5b16e8511eac7090858c9b3f29af0a2ac6a/warehouse/observations/tasks.py#L90

Reply:
ah, thanks definitely felt like I was holding it wrong.

Reply:
281781fa9